util/threadpool_imp.cc

```diff
@@ -110,6 +142,10 @@ struct ThreadPoolImpl::Impl {
 
  int total_threads_limit_;
  std::atomic_uint queue_len_;  // Queue length. Used for stats reporting
+ // Number of reserved threads by other tasks in bgthreads_
+ int reserved_threads_;
```

Comment:
nit: num_reserved_threads_ to be consistent with below

Reply:
"other tasks" sounds a bit random/unclear, although I understand it refers to tasks that reserve threads (e.g, subcompaction) as opposed to tasks do not reserve threads (e.g, flush) because I was exposed to this PR's motivation. 

Is there a way to better clarify the comment so that people w/o much context of your PR understand it?

Reply:
I see your point. I will revise it accordingly.